features/cellular/framework/AT/AT_CellularDevice.h

```diff
@@ -165,6 +206,9 @@ class AT_CellularDevice : public CellularDevice {
 private:
     void urc_nw_deact();
     void urc_pdn_deact();
+
+private:
+    const intptr_t *_property_array;
```

Comment:
`_property_array` is not initialized in `AT_CellularDevice `constructor causing "TestAT_CellularDevice, Create" UT test to fail.

Reply:
Fixed now